linz_logger/test_logger.py

```diff
@@ -28,3 +29,15 @@ def test_trace_at_debug_level(capsys):
     get_log().trace("abc")
     stdout, _ = capsys.readouterr()
     assert stdout == ""
+
+
+def test_timestamp(capsys):
+    """Test timestamp is actually a timestamp"""
+    set_level(LogLevel.trace)
+
+    systime = int(os.popen("date +%s").read()) * 1000
+    get_log().trace("abc")
+    stdout, _ = capsys.readouterr()
+    log = json.loads(stdout)
+
+    assert log["time"] - systime < 1000
```

Comment:
Any kind of time-dependent test like this is going to be brittle. The common way to fix that would be to sample the time and dependency-injecting a static clock into `get_log` to verify that the timestamp is the same.

Reply:
if this test takes more than a second we have bigger problems.

Reply:
In this case it might be fine - it's not like this is ever going to be a huge code base. But you'd be surprised.